server/verify-brackets.js

```diff
@@ -0,0 +1,18 @@
+const config = require('server/routes/config');
+
+const isReachable = require('is-reachable');
+var brambleServer = config.editorHOST;
+
+console.log("Checking for Bramble Server. " + brambleServer);
+
+isReachable(brambleServer).then(reachable => {
+  
+  if(reachable)
```

Comment:
In JS you want to always put the `{` on the same line as the statement.  There are some weird bugs you can get into with `return` and automatic semi-colon insertion, for example:

```js
return // actually returns `undefined`
{
  data: 1
};

// vs. 

return { // does what you want
  data: 1
};
```

Reply:
Oh I didn't even know that could happen with JS. Good to know.